enumfields/enums.py

```diff
@@ -15,6 +16,9 @@ def __new__(mcs, name, bases, attrs):
             if hasattr(attrs, '_member_names'):
                 attrs._member_names.remove('Labels')
 
+        if sys.version_info > (3, 9, 2):
```

Comment:
Is the `>` correct? Should it be `>=`?

Reply:
`sys.version_info` equals `(3, 9, 2, 'final', 0)`, this greater `(3, 9, 2)`